book/src/themes.md

```diff
@@ -205,32 +205,32 @@ These scopes are used for theming the editor interface.
-| `ui.linenr.selected`     |                                     |
-| `ui.statusline`          | Statusline                          |
-| `ui.statusline.inactive` | Statusline (unfocused document)     |
-| `ui.popup`               |                                     |
-| `ui.popup.info`          |                                     |
-| `ui.window`              |                                     |
-| `ui.help`                |                                     |
-| `ui.text`                |                                     |
-| `ui.text.focus`          |                                     |
-| `ui.text.info`           |                                     |
-| `ui.menu`                |                                     |
-| `ui.menu.selected`       |                                     |
-| `ui.selection`           | For selections in the editing area  |
-| `ui.selection.primary`   |                                     |
-| `warning`                | Diagnostics warning (gutter)        |
-| `error`                  | Diagnostics error (gutter)          |
-| `info`                   | Diagnostics info (gutter)           |
-| `hint`                   | Diagnostics hint (gutter)           |
-| `diagnostic`             | For text in editing area            |
+| Key                      | Notes                                          |
+| ---                      | ---                                            |
+| `ui.background`          |                                                |
+| `ui.cursor`              |                                                |
+| `ui.cursor.insert`       |                                                |
+| `ui.cursor.select`       |                                                |
+| `ui.cursor.match`        | Matching bracket etc.                          |
+| `ui.cursor.primary`      | Cursor with primary selection                  |
+| `ui.linenr`              | Line numbers                                   |
+| `ui.linenr.selected`     | Line number for the line the cursor is on      |
+| `ui.statusline`          | Statusline                                     |
+| `ui.statusline.inactive` | Statusline (unfocused document)                |
+| `ui.popup`               | Documentation popups (e.g space-k)             |
+| `ui.popup.info`          | prompt for multiple key options                |
+| `ui.window`              | Border lines separating splits                 |
+| `ui.help`                | Description box for commands                   |
+| `ui.text`                | command prompts, popup text, etc.              |
+| `ui.text.focus`          |                                                |
+| `ui.text.info`           | the key: command text in `ui.popup.info` boxes |
+| `ui.menu`                | code and command completion menus              |
+| `ui.menu.selected`       | selected autocomplete item                     |
```

Comment:
This is a bit of a nit but would you mind capitalizing the first characters? Just for consistency's sake in this block.

Reply:
oh, yeah will do.

Reply:
Done!